sdk/web-pubsub/web-pubsub-express/src/cloudEventsDispatcher.ts

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CloudEvent, Message, HTTP } from "cloudevents";
+import { IncomingMessage, ServerResponse } from "http";
+import { URL } from "url";
+import {
+  ConnectRequest,
+  ConnectResponse,
+  UserEventRequest,
+  DisconnectedRequest,
+  ConnectedRequest,
+  ConnectionContext,
+  ConnectResponseHandler,
+  UserEventResponseHandler,
+  WebPubSubEventHandlerOptions
+} from "./cloudEventsProtocols";
+
+class DefaultConnectResponseHandler implements ConnectResponseHandler {
+  constructor(private response: ServerResponse) {}
+  public success(response?: ConnectResponse): void {
+    this.response.statusCode = 200;
+    this.response.setHeader("Content-Type", "application/json; charset=utf-8");
+    this.response.end(JSON.stringify(response));
+  }
+  public fail(code: 400 | 401 | 500, detail?: string): void {
+    this.response.statusCode = code;
+    this.response.end(detail ?? "");
+  }
+}
+
+class DefaultUserEventResponseHandler implements UserEventResponseHandler {
+  constructor(private response: ServerResponse) {}
+  public success(data?: string | ArrayBuffer, dataType?: "binary" | "text" | "json"): void {
+    this.response.statusCode = 200;
+    switch (dataType) {
+      case "json":
+        this.response.setHeader("Content-Type", "application/json; charset=utf-8");
+        break;
+      case "text":
+        this.response.setHeader("Content-Type", "text/plain; charset=utf-8");
+        break;
+      default:
+        this.response.setHeader("Content-Type", "application/octet-stream");
+        break;
+    }
+    this.response.end(data ?? "");
+  }
+  public fail(code: 400 | 401 | 500, detail?: string): void {
+    this.response.statusCode = code;
+    this.response.end(detail ?? "");
+  }
+}
+
+export class CloudEventsDispatcher {
```

Comment:
There aren't docs on this or any of the methods, could we add some? I'm also not sure if this is meant to be exported publicly. If not, I think we need to add the `@internal` doc tag to anything that's exported from any file so it doesn't show up in docs. (I'm not sure if the story around that has changed though)

Reply:
hm.. it's not exported publicly, the api-extract seems can auto detect if it is exported from index.ts. So that's why it does not show in `web-pubsub-express.api.md`.

Reply:
Does reference doc use another logic that it needs `@internal` doc to hide it? Will add